### PR TITLE
[0050] Amended the rollover documentations

### PR DESCRIPTION
**Pull request by defong**
### Context
The rollover period

### Changes proposed in this pull request
Added a sequence diagram to depict the whole before and after rollover period. 
Amended with more details towards the whole rollover period.

### Guidance to review
Perform this locally and use time machine **wisely**

```bash
    bundle exec rake 'rollover:create_recruitment_cycle[2023, "2022-10-14", "2023-09-30"]'
    bundle exec rake rollover:providers
```

Have `find` setup

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by defong** [comment]
[Rollover - DRY run](https://trello.com/c/5gJdXCPb/50-rollover-dry-run)

**Review by goodviber — APPROVED** [review]
Lgtm, not that I know everything about rollover...

File: docs/rollover.md [hunk 1]
```diff
@@ -1,67 +1,171 @@
...
-3. Complete the steps on Publish. You'll need to have run the above rake tasks
-  before making any setting changes on Publish.
+2. Merge the **Start Rollover PR**
 
 ## During Rollover
 
 1. Create an **End Rollover PR** including the following code changes:
+    - Update the setting for `can_edit_current_and_next_cycles: false`
+    - Replace `google_forms: current_cycle:` settings with those in
+      `google_forms: next_cycle:`
```
Comment (defong) [inline docs/rollover.md:161]: TBC

File: docs/rollover.md [hunk 2]
```diff
@@ -1,67 +1,171 @@
...
+    ```bash
+    bundle exec rake rollover:providers
+    ```
+3. Create **new Google forms** for adding PE courses for the next cycle and
+  update the links in the settings `google_forms: next_cycle`.
 
-2. Rollover providers by running:
+4. Create a **Start Rollover PR** including the following code changes:
+    - Update the setting for `can_edit_current_and_next_cycles: true`
+    - Update the links in the settings `google_forms: next_cycle`
```
Comment (defong) [inline docs/rollover.md:147]: TBC

Comment (defong) [reply docs/rollover.md:147]: to resolve another time

File: docs/rollover.md (hunk 1 above)
Comment (defong) [reply docs/rollover.md:161]: to resolve another time

**Pull request merged**